ui/src/pages/frontsite/PatronProfile/PatronCurrentLoans/components/ExtendButton/ExtendButton.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
+import { Button, Icon, Popup } from 'semantic-ui-react';
+import { invenioConfig, ES_DELAY } from '@config';
+import _get from 'lodash/get';
+import _has from 'lodash/has';
+
+export const INFO_MESSAGES = {
```

Comment:
minor: Do you need this export? I see you use it only for tests so how about defining these messages as `static` inside the `ExtendButton` component? Or otherwise in separate file `messages.js` inside the folder?